scripts/vit-watchdog

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+#
+# Simple wrapper to reload vit upon task changes
```

Comment:
The description feels a little vague.

Perhaps something more like this?

```python
# Watchdog script for vit -- automatically reloads vit when task data changes.
# Monitors the pending.data file and sends a SIGUSR1 signal to vit to trigger a refresh.
```

Reply:
Also, it just occurred to me that the comment should mention that this will ONLY work with Taskwarrior < 3.0

Reply:
done.